Smarty/templates/sliderwidget.tpl

```diff
@@ -15,6 +15,7 @@
  *************************************************************************************************/
 -->*}
 <script src="include/components/Carousel/vanilla-js-carousel.min.js"></script>
+<script src="include/components/JSZip/JSZip.min.js"></script>
```

Comment:
I would prefer that we avoid this extra library. Can we get that zipped information from coreBOS?  I see below that you are fetch(ing) a URL which I suppose is to the coreBOS install to get the images, I would prefer that coreBOS sent it already compressed. Not only do we have all that functionality already in coreBOS but we will optimize the bandwidth and local browser memory.